app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :likes
+  has_many :comments
+
+  after_save :update_user_post_counter
+
+  def return_last_five_comments
+    comments.last(5)
+  end
```

Comment:
- [ ] It should return the 5 most recent comments for a given post in descending order.

Hint:
`comments.order('created_at Desc').limit(5)`

Reply:
- [ ] Same things go here for the **hints**.